pkg/kapis/devops/v1alpha2/register.go

```diff
@@ -59,6 +64,11 @@ func AddToContainer(container *restful.Container, ksInformers externalversions.S
 		return err
 	}
 
+	err = AddJenkinsToContainer(ws, endpoint)
```

Comment:
If endpoint is empty, means Devops is not enabled, `AddJenkinsToContainer` will return an error

Reply:
fixed